get_alphafold_cifs.sh

```
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+ACCESS_DATE=$(date +"%Y-%m-%d")
+
+# Create a directory for downloading the alphafold structures
+AF_DIR="af_cifs"
+mkdir -p "./examples/"$AF_DIR
+cd "./examples/"$AF_DIR
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)